StatisticsInR/2.1_simple_linear_regression.Rmd

```diff
@@ -0,0 +1,1223 @@
+output from `lm()`. This function gives a big table with lots of information.
+The first line shows the model formula used for the model object. The second line shows
+a summary of the residuals of the model and the
+standard errors are shown as the second column in the third part, `Coefficients:`.
+
+```{r, include = T, echo =T}
+summary(ModelObject)
+```
+</br>
+
+If we take the `summary()` of the example model, we can see the standard error or the 
+intercept ($\alpha$) is 0.95825, and the standard error for the slope ($\beta$) is 
+0.05483.
+
+</br>
+
+##### Confidence intervals
+
+For interpretation of the uncertainty, it can be easier to use the standard error
+to calculate confidence intervals (CI). Confidence intervals indicate the range of plausible values for a parameter. **They represent an interval, that if you were to collect a sample and run the
+analysis, then
+repeat that many many times AND each time
+draw a confidence interval, on average 95% of the time, the true population value of the parameter would be found in within the confidence interval.** 
+
+If you need a reminder of this click [here](link to uncertainty page).
+
+</br>
+
+The confidence interval can be calculated using this formula:
+
+$$
+\begin{aligned}
+UpperCI = estimate + (1.96 SE) \\
+LowerCI = estimate - (1.96 SE) \\
+\end{aligned}
+$$
+1.96 is used because in a standard normal distribution 95% of the distribution lies 
+within 1.96 standard deviations of the mean. In this case the distribution is the sampling 
+distribution, which is normal for $\alpha$ and $\beta$ and the standard deviation is the standard error. 
+
+</br>
+
+Using the formulas above, calculate the confidence intervals for the intercept and
+slope from the example model.
+
+</br>
+
+<details><summary> I had a go at calculating, what is the correct answer?</summary>
+
+```{r, include = T, echo =T}
+# INTERCEPT
+
+# Upper confidence interval
+UpperCI_intercept <- summary(ModelObject)$coefficients[1,1] + 
+  (1.96*summary(ModelObject)$coefficients[1,2])
+# Lower confidence interval
+LowerCI_intercept <- summary(ModelObject)$coefficients[1,1] - 
+  (1.96*summary(ModelObject)$coefficients[1,2])
+
+# Print the interval
+c(UpperCI_intercept, LowerCI_intercept) 
+
+# SLOPE
+
+# Upper confidence interval
+UpperCI_slope <- summary(ModelObject)$coefficients[2,1] + (1.96*summary(ModelObject)$coefficients[2,2])
+# Lower confidence interval
+LowerCI_slope <- summary(ModelObject)$coefficients[2,1] - (1.96*summary(ModelObject)$coefficients[2,2])
+
+# Print the interval
+c(UpperCI_slope, LowerCI_slope) 
+```
+
+</details>
+
+</br>
+
+It is also possible to get R to calculate the confidence intervals for you. To do this
+you can use the `confint()` function. The argument is a model object. 
+
+```{r, include = T, echo = T}
+confint(ModelObject)
+```
+
+</br>
+
+Hopefully these confidence intervals look the same as those you calculated yourself. 
+
```

Comment:
Maybe this should be an actual learnr excesice.

Reply:
This would be good, looking at some other examples, does this mean the whole page would need to be a learnr output? Would this mess up the shiny app?